test/Core.Test/Billing/Services/OrganizationBillingServiceTests.cs

```diff
@@ -353,4 +354,97 @@ await sutProvider.GetDependency<IStripeAdapter>()
     }
 
     #endregion
+
+    [Theory, BitAutoData]
+    public async Task UpdateOrganizationNameAndEmail_UpdatesStripeCustomer(
+        Organization organization,
+        SutProvider<OrganizationBillingService> sutProvider)
+    {
+        organization.Name = "Short name";
```

Comment:
Also non-blocker: [same](https://github.com/bitwarden/server/pull/6599/files#r2548034430) but with a different name for this case.